src/Dom/Browser.Dom.fs

```diff
@@ -808,6 +808,32 @@ type [<AllowNullLiteral>] TreeWalkerType =
 type [<AllowNullLiteral>] BarProp =
     abstract visible: bool with get, set
 
+[<StringEnum; RequireQualifiedAccess>]
+type ScreenOrientationType =
+    | [<CompiledName("portrait-primary")>] PortraitPrimary
+    | [<CompiledName("portrait-secondary")>] PortraitSecondary
+    | [<CompiledName("landscape-primary")>] LandscapePrimary
+    | [<CompiledName("landscape-secondary")>] LandscapeSecondary
+
+[<StringEnum; RequireQualifiedAccess>]
+type OrientationLockType =
+    | [<CompiledName("any")>] Any
+    | [<CompiledName("natural")>] Natural
+    | [<CompiledName("landscape")>] Landscape
+    | [<CompiledName("portrait")>] Portrait
+    | [<CompiledName("portrait-primary")>] PortraitPrimary
+    | [<CompiledName("portrait-secondary")>] PortraitSecondary
+    | [<CompiledName("landscape-primary")>] LandscapePrimary
+    | [<CompiledName("landscape-secondary")>] LandscapeSecondary
+
+type [<AllowNullLiteral>] ScreenOrientation =
+    inherit EventTarget
+    abstract ``type``: ScreenOrientationType
+    abstract angle: int
```

Comment:
According to MDN this is an unsigned short, but `int` is probably more convenient :+1:

Reply:
Sorry, I didn't know any equivalent of `ushort` in F#, so went with `int`